app/v1/controllers/requestParsers/validators/RetrievePeriodSummaryValidator.scala

```diff
@@ -32,7 +32,8 @@ class RetrievePeriodSummaryValidator extends Validator[RetrievePeriodSummaryRawD
     List(
       NinoValidation.validate(data.nino),
       BusinessIdValidation.validate(data.businessId),
-      PeriodIdValidation.validate(data.periodId)
+      PeriodIdValidation.validate(data.periodId),
+      data.taxYear.map(TaxYearValidation.validate).getOrElse(Nil)
```

Comment:
We might also need to validate against a minimum tax year (2024)? See changes added to [this commit](https://github.com/hmrc/self-employment-business-api/pull/133/commits/764a6e48089a8e57a75888ce9601b86dac93f02f) for the List endpoint. (Might need to merge that first?)

Reply:
The other PR is merged into main so the validation code can be used (if we want to do this validation).